packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -573,7 +612,10 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with AutomaticK
 
   void _handleSelectionChanged(TextSelection selection, SelectionChangedCause cause) {
     if (cause == SelectionChangedCause.longPress) {
-      _editableTextKey.currentState?.bringIntoView(selection.base);
+      _editableText?.bringIntoView(selection.base);
+    }
+    if (_shouldShowHandles(cause)) {
+      _editableText.showHandles();
```

Comment:
Is this an API breaking change? i.e. if I previously used EditableText as is, it won't show handles anymore after I flutter upgrade?